tests/test_integration.py

```diff
@@ -46,4 +39,9 @@ def test_index_page(monkeypatch, client):
     response = client.get('/')
```

Comment:
Instead of the monkeypatching before this, the suggestion is that we directly add an item into our fake MongoDB, for example with the below - though I warn untested!
```
def test_index_page(client):
    client = pymongo.MongoClient(os.getenv('MONGO_CONNECTION_STRING'))
    db = client[os.getenv('MONGO_DATABASE')]
    collection = db['Items']
    newItem = {"title": "wash the floor", "status": "To Do"}
    collection.insert_one(newItem)
    # Make a request to our app's index page
    response = client.get('/')

    assert response.status_code == 200

    assert 'wash the floor' in response.data.decode()
   
```